jenkins/files/Debian/jenkins.conf

```diff
@@ -1,38 +1,40 @@
-{% from "jenkins/map.jinja" import jenkins with context %}
-# defaults for jenkins continuous integration server
+{%- from "jenkins/map.jinja" import jenkins with context -%}
+# defaults for Jenkins automation server
 
 # pulled in from the init script; makes things easier.
 NAME=jenkins
 
-# location of java
-JAVA={{ jenkins.java_executable }}
-
 # arguments to pass to java
-#JAVA_ARGS="-Djava.awt.headless=true"  # Allow graphs etc. to work even when an X server is present
+
+# Allow graphs etc. to work even when an X server is present
+#JAVA_ARGS="-Djava.awt.headless=true"
+
 #JAVA_ARGS="-Xmx256m"
-#JAVA_ARGS="-Djava.net.preferIPv4Stack=true" # make jenkins listen on IPv4 address
+
+# make jenkins listen on IPv4 address
+#JAVA_ARGS="-Djava.net.preferIPv4Stack=true"
 
 JAVA_ARGS="{{ jenkins.java_args }}"
 
-PIDFILE=/var/run/${NAME}/${NAME}.pid
+PIDFILE=/var/run/$NAME/$NAME.pid
 
 # user and group to be invoked as (default to jenkins)
 JENKINS_USER="{{ jenkins.user }}"
 JENKINS_GROUP="{{ jenkins.group }}"
 
 # location of the jenkins war file
-JENKINS_WAR=/usr/share/${NAME}/${NAME}.war
+JENKINS_WAR=/usr/share/$NAME/$NAME.war
```

Comment:
why the change here? 
It's a common practice to put variables in curly braches for shell expansion as well as surround with quotes (missing here) to prevent issues with spaces